api.py

```
@@ -1,4 +1,4 @@
-from fastapi import FastAPI, File, UploadFile, HTTPException, Query
+from fastapi import FastAPI, File, UploadFile, HTTPException, Query, Body
```

Ruff (0.11.9)

`fastapi.Query` imported but unused

Remove unused import: `fastapi.Query`

(F401)